templates/workspaces/base/porter.yaml

```diff
@@ -1,10 +1,18 @@
 ---
 name: tre-workspace-base
-version: 0.1.14
+version: 0.2.0
 description: "A base Azure TRE workspace"
 registry: azuretre
 
 credentials:
+  # Credentials for interacting with the AAD Auth tenant
+  - name: auth_client_id
+    env: AUTH_CLIENT_ID
+  - name: auth_client_secret
+    env: AUTH_CLIENT_SECRET
+  - name: auth_tenant_id
+    env: AUTH_TENANT_ID
+  # Credentials for interacting with Azure
```

Comment:
If these values are not present can we still deploy?

Reply:
These values are passed in by the RP and locally they are passed in by `make bundle-install`

Reply:
Will they always exist? Even if people don't want to automatically create app reg? I guess they will, just might not have the correct permissions.